pkg/virtctl/imageupload/imageupload.go

```diff
@@ -153,6 +153,9 @@ func (c *command) run(cmd *cobra.Command, args []string) error {
 		if !(k8serrors.IsNotFound(err) && !noCreate) {
 			return err
 		}
+		if !noCreate && len(pvcSize) == 0 {
+			return fmt.Errorf("When create pvc, the size must be specified")
```

Comment:
just a nit, could you change to "When creating PVC, the size must be specified"?